packages/manager/modules/vrack-services/src/onboarding/controller.js

```diff
@@ -1,7 +1,30 @@
-import { GUIDES } from './constants';
+import { GUIDES, DISCOVER_CTA, TRACKING_NAME } from './constants';
 
 export default class VrackServicesOnboardingCtrl {
-  constructor() {
-    this.GUIDES = GUIDES;
+  /* @ngInject */
+  constructor($translate, coreConfig, atInternet, $window) {
+    const { ovhSubsidiary } = coreConfig.getUser();
+    this.GUIDES = GUIDES.map((guide) => ({
+      ...guide,
+      title: $translate.instant(guide.title),
+      description: $translate.instant(guide.description),
+      link: guide.links[ovhSubsidiary] || guide.links.DEFAULT,
+    }));
```

Comment:
**suggestion:** This can be moved to `$onInit` to make construction of controller light as this delays the initialisation of controller